### PR TITLE
remove command UX enhancement

### LINKED ISSUE #93: Improve remove index's tooltip and parameter type
## Rationale
Currently, when selecting a track to be removed from the queue, it isn't immediately obvious how it is going to work.

## Description

The UX of selecting the track to be removed could be improved with a tooltip that better clarifies how the queue indexing works. Additionally, the index parameter could be converted to an `INTEGER` option.

## Reference
[Application Command Option Type](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-type)

### PR DESCRIPTION
**Pull request by afonsojramos**
Closes #93.

* Changed the tooltip:
image

* Added handling support for negative numbers

### CODE REVIEW

File: src/commands/remove.rs [hunk 1]
```diff
@@ -22,29 +22,27 @@ pub async fn remove(
...
         .value
         .as_ref()
         .unwrap()
-        .as_u64()
-        .unwrap() as usize;
+        .as_i64()
+        .unwrap();
 
+    if signed_remove_index.is_negative() || signed_remove_index <= 0 {
+        return create_response(&ctx.http, interaction, "Please provide an index >= 1!").await;
```
Comment (rafaeldamasceno) [inline src/commands/remove.rs:35]: This is just a nitpick so feel free to disregard, but I think having the inequality in writing (`greater than or equal to`) would be preferable.

**Review by rafaeldamasceno — COMMENTED** [review]
I left some comments about the wording of the bot messages.

File: src/events/serenity_handler.rs [hunk 2]
```diff
@@ -81,7 +81,7 @@ impl EventHandler for SerenityHandler {
                         .create_option(|option| {
                             option
                                 .name("index")
-                                .description("Position of the track (0 is currently playing)")
+                                .description("Position of the track (next track is 1)")
```
[the remark this answers is not in the document]
Comment (afonsojramos) [reply src/events/serenity_handler.rs:84]: Agreed here!

File: src/commands/remove.rs (hunk 1 above)
Comment (afonsojramos) [reply src/commands/remove.rs:35]: I think I disagree, but it is not a hard disagree 🤔 Everyone understands math, it's not like the users do not understand a simple `>= 1` in my opinion.

File: src/commands/remove.rs [hunk 3]
```diff
@@ -22,29 +22,27 @@ pub async fn remove(
...
         .unwrap()
         .value
         .as_ref()
         .unwrap()
-        .as_u64()
-        .unwrap() as usize;
+        .as_i64()
+        .unwrap();
 
+    if signed_remove_index.is_negative() || signed_remove_index <= 0 {
```
Comment (joao-conde) [inline src/commands/remove.rs:34]: Just:

```rust
if !signed_remove_index.is_positive()
```

**Review by joao-conde — CHANGES_REQUESTED** [review]

**Review by joao-conde — APPROVED** [review]

**Pull request merged**